rpmlint/pkg.py

```diff
@@ -950,6 +952,9 @@ def cleanup(self):
         if self.dirname:
             getstatusoutput(('rm', '-rf', self.dirname))
 
+    def files(self):
+        return self._files
+
```

Comment:
why not access the variable directly?

Reply:
Because I want to follow the same interface as in `class Pkg`, where you also use `pkg.files()['my_file']`.

Reply:
It is not really needed pkg.files['whatever'] works just fine,the pkg.py still needs these cleanups really.

Reply:
Oth it can stay and be fixed in one huge chunk, okay...